app/editor/src/features/admin/sources/DataSourceActions.tsx

```diff
@@ -0,0 +1,44 @@
+import { FormikCheckbox } from 'components/formik';
+import { getIn, useFormikContext } from 'formik';
+import { useNamespace } from 'hooks';
+import { IDataSourceModel, ISourceActionModel } from 'hooks/api-editor';
+import React from 'react';
+import { useLookup } from 'store/hooks';
+
+import * as styled from './styled';
+
+export const DataSourceActions: React.FC = (props) => {
```

Comment:
This placed checkboxes for each action.  I find this very slow however, it's visible to the user.  It shouldn't be so slow.